x/gov/tokenbalance/tokenbalance.go

```diff
@@ -0,0 +1,66 @@
+package stakingtally
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	govkeeper "github.com/cosmos/cosmos-sdk/x/gov/keeper"
+	"github.com/cosmos/cosmos-sdk/x/gov/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+)
+
+// NewTokenBalanceTallyHandler creates a new governance tally strategy for a giving voting to token balances for a particular denom
+func NewTokenBalanceTallyHandler(gk govkeeper.Keeper, bk govtypes.BankKeeper, denom string) govtypes.TallyStrategy {
+	return func(ctx sdk.Context, proposal govtypes.Proposal) (passes bool, burnDeposits bool, tallyResults govtypes.TallyResult) {
+		return handleTokenBalanceTally(ctx, proposal, gk, bk, denom)
+	}
+}
+
+// TODO: Break into several smaller functions for clarity
+
+// Tally iterates over the votes and updates the tally of a proposal based on the voting power of the
+// voters
+func handleTokenBalanceTally(ctx sdk.Context, proposal govtypes.Proposal, gk govkeeper.Keeper, bk govtypes.BankKeeper, denom string) (passes bool, burnDeposits bool, tallyResults govtypes.TallyResult) {
```

Comment:
Where should we put this if we move staking tally to staking module ?
Maybe bank module? :)

Reply:
Hmm, I guess.  Is it going to be odd to have a reference to the gov module within the bank module, even if its just in a submodule?

Should we maybe have a separate folder of tallystrategies, and have that filled with submodules?